RecoMuon/GlobalTrackingTools/src/GlobalMuonRefitter.cc

```diff
@@ -817,25 +826,25 @@ vector<Trajectory> GlobalMuonRefitter::transform(const reco::Track& newTrack,
     }
   }
 
-/*  
+/*
```

Comment:
All the commented out lines below could be simply removed, or replaced with some LogDebug/LogTrace